src/utils/importPaths.ts

```diff
@@ -0,0 +1,103 @@
+const CAIRO_COMMON_PATH = ['starkware', 'cairo', 'common'];
```

Comment:
It look good to me. The only thing I would change and it is a nitpick is naming of some constants:
* const which uses warp memory method I would prefix them with `WM_`